todo_app/app.py

```diff
@@ -1,102 +1,43 @@
 from flask import Flask, render_template, request, redirect
 from todo_app.todo import Item
-
 from todo_app.flask_config import Config
-import requests
-import os
 from todo_app.view_model import ViewModel
+from todo_app.todomongo import add_todo_item, items, update_status, delete_item
 
 def create_app():
     app = Flask(__name__)
     app.config.from_object(Config())
 
 
     @app.route('/')
-    def index():
-        
-        url = f"https://api.trello.com/1/boards/{os.getenv('TRELLO_BOARD_ID')}/lists"
-
-        querystring = {
-                "key":os.getenv("TRELLO_API_KEY"),
-                "token":os.getenv("TRELLO_API_TOKEN"),
-                "cards": "open"
-        }
 
-        response = requests.get(url, params=querystring)
-
-        response_json = response.json()
-
-        items = []
-
-        for trello_list in response_json:
-            for card in trello_list['cards']:
-                myItem = Item(card['id'] , card['name'] , trello_list['name'])
-                items.append(myItem)
+    def index():
 
-        item_view_model = ViewModel(items)
+        item_view_model = ViewModel(items())
         return render_template('index.html',
         view_model=item_view_model)
 
-    @app.route('/add_card', methods=['POST'])
-    def add_new_card():
-
-        url = "https://api.trello.com/1/cards"
+    @app.route('/add_task', methods=['POST'])
+    def add_new_item():
 
         title = request.form['todo_title']
+        add_todo_item(title)
```

Comment:
I think this approach is quite a bit cleaner than your previous one with all the logic in this file - nice 😄